portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -17,16 +17,22 @@
 /** Class containing comment in a website */
 public final class Comment {
 
+  private final long id;
   private final String name;
   private final String likedGame;
   private final String comment;
 
-  public Comment(String name, String likedGame, String comment){
+  public Comment(long id, String name, String likedGame, String comment){
+      this.id = id;
       this.name = name;
       this.likedGame = likedGame;
       this.comment = comment;
   }
 
+  public long getId(){
```

Comment:
Is this actually read anywhere in client? If not, I think it's better to not add fields which aren't read.